internal/cli/kraft/run/runner_kernel.go

```diff
@@ -75,6 +77,14 @@ func (runner *runnerKernel) Prepare(ctx context.Context, opts *RunOptions, machi
 		}
 	} else {
 		machine.Spec.Architecture = opts.Architecture
+
+		if opts.Architecture != "x86_64" &&
+			opts.Architecture != "arm" &&
+			opts.Architecture != "arm64" {
```

Comment:
Let's put this in a method returning a slice (similar to [this](https://github.com/unikraft/kraftkit/blob/staging/machine/platform/platform.go#L44)) inside of https://github.com/unikraft/kraftkit/blob/staging/unikraft/arch/host.go since this file contains known info about supported Unikraft architectures.
And use:
```go
import (
  "slices"
  ukarch "kraftkit.sh/unikraft/arch"
)
// [...]
if !slices.Contains[[]string](ukarch.KnownArchs(), opts.Architecture) {
```
See [`slices.Contains`](https://pkg.go.dev/slices#Contains)